Dockerfile

```diff
@@ -51,6 +48,23 @@ RUN cd /serialbox && \
     make install && \
     /bin/rm -rf /serialbox
 
+# Install Google Cloud
+RUN apt-get update && apt-get install -y  apt-transport-https ca-certificates gnupg curl gettext
+
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list &&\
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+
+
+RUN apt-get update && apt-get install -y google-cloud-sdk jq python3-dev python3-pip kubectl gfortran
+
+# install some python packages
+RUN pip install numpy zarr xarray ipython pyyaml dask netCDF4 rechunker pytest && \
+    pip install dask[array] --upgrade
+
+# Zarr conversion
+COPY serial_convert /serial_convert
```

Comment:
You copy it here but you also mount it below in `enter.sh`. Any particular reason?